test/avocado/cockpit.py

```diff
@@ -21,8 +21,9 @@
 import re
 import subprocess
 import shutil
-
```

Comment:
Are the test/avocado/ changes really dependent on the reorganization of bots/machine? Ideally the latter should be completely independent and backwards compatible - first, that's easier to review, and second, this is API for third-party projects, so it should not break API. Maybe this happened during over-squashing the  commits?

Can you please clean up the commit message a bit, prefix it with  "bots:", removing the "fixup"s, and try to split the bots/ from the tests/ changes?

Reply:
@martinpitt it was highly dependent, but after you merged it partially to #10544, changes are now independent. so that I can split it into two PRs